### PR TITLE
[web] allow imports to line-break

### PR DESCRIPTION
**Pull request by yjbanov**
We are about to `dart format` all the Dart code in the repo, which line-breaks some of our imports.

This PR updates `sdk_rewriter.dart` to support line-broken imports.

### CODE REVIEW

**Review by goderbauer — APPROVED** [review]
LGTM

File: web_sdk/sdk_rewriter.dart [hunk 1]
```diff
@@ -79,6 +79,7 @@ List<Replacer> generatePartsPatterns(String libraryName, bool isPublic) {
     // Remove library directives.
     AllReplacer(RegExp(r'\nlibrary .+;'), ''),
     // Remove imports/exports from all part files.
+    AllReplacer(RegExp(r"\nimport '.+'\n\s*if \(dart.library.html\) '.*';"), ''),
```
Comment (ditman) [inline web_sdk/sdk_rewriter.dart:82]: can we not hardcode `dart.library.html` or maybe also allow for `dart.library.js_interop` as well, so our code doesn't mysteriously break when we migrate the code to not referencing dart.library.html? Bugs in the sdk rewriter are very hard to pinpoint :/

Comment (yjbanov) [reply web_sdk/sdk_rewriter.dart:82]: Done.

**Comment by goderbauer** [comment]
Looks like something is still failing with the line breaks…

**Review by mdebbar — APPROVED** [review]

**Comment by mdebbar** [comment]
@yjbanov I think we need to remove this restriction now: https://github.com/flutter/engine/blob/f0ff4f25bda8c249e7bef1cff54b268e36894891/web_sdk/test/js_access_test.dart#L184-L187

**Comment by yjbanov** [comment]
Yep, fixed.

File: web_sdk/sdk_rewriter.dart [hunk 2]
```diff
@@ -79,6 +79,7 @@ List<Replacer> generatePartsPatterns(String libraryName, bool isPublic) {
     // Remove library directives.
     AllReplacer(RegExp(r'\nlibrary .+;'), ''),
     // Remove imports/exports from all part files.
+    AllReplacer(RegExp(r"\nimport '.+'\n\s*if \(dart\.library\..*\) '.*';"), ''),
```
Comment (ditman) [inline web_sdk/sdk_rewriter.dart:82]: Not that it matters too much, but I'd do a simpler regex to handle both `import` cases?

https://regexr.com/8a1l1

?

**Pull request merged**